src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -300,6 +322,9 @@ def execute(self, emulator=None):
         else:
             self._setStatus(returnCode, returnMessage)
 
+        # update FJR report info with subprocess info
+        self.report.updateSubprocessInfo(sysTime, userTime, startTime, endTime)
```

Comment:
I think it's safer to move these 2 lines under the try/except block below.

Reply:
done